src/mcts/search.cc

```diff
@@ -1028,7 +1028,7 @@ SearchWorker::NodeToProcess SearchWorker::PickNodeToExtend(
 
     if (second_best_edge) {
       int estimated_visits_to_change_best = best_edge.GetVisitsToReachU(
-          second_best, puct_mult, fpu, params_.GetLogitQ());
+          second_best, puct_mult, fpu, even_draw_score, params_.GetLogitQ());
```

Comment:
I don't think even_draw_score is the right value to use here...  Should be just draw_score I think?

Reply:
It should be draw_score. Looks like you already fixed it, so I'll close this PR.